src/modules/meta_producto/meta_producto.service.ts

```
@@ -1,673 +1,726 @@
+
+        return {
+          status: false,
+          message: 'No se detectaron cambios en el meta producto',
+          error: 'Sin cambios',
+          data: [result],
+        };
+      }
+
+      // Actualizar meta_producto solo si hay cambios básicos
+      if (hasBasicChanges) {
+        const { data: updatedData, error: updatedError } =
```

ESLint

[error] 'updatedData' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)